mobile/lib/constants/config.dart

```diff
@@ -14,7 +14,7 @@ class Config {
   static double get minimumTextScaleFactor => 1.0;
   static double get maximumTextScaleFactor => 1.1;
 
-  static String get airqoApi => 'https://api.airqo.net/api';
+  static String get airqoApi => 'https://platform.airqo.net/api';
```

Comment:
Hi @BenjaminSsempala why have we changed this??

Reply:
I'd changed it because I was getting bad getways on all the requests until i changes it to platform @NoahNsimbe